src/Turing.jl

```diff
@@ -8,6 +8,7 @@ using Libtask
 using Tracker: Tracker
 
 import AdvancedVI
+using DynamicPPL: DynamicPPL, LogDensityFunction
 import DynamicPPL: getspace, NoDist, NamedDist
 import LogDensityProblems
```

Comment:
Maybe this can be removed now?
```suggestion
```

Reply:
Still need it for all the optimation related stuff.